code/qcommon/cm_load.c

```diff
@@ -748,6 +748,9 @@ int CM_LoadSubBSP(const char *name)
 
     CM_FloodAreaConnections(cm);
 
+    // write the BSP name into the clipmap struct so that it could be reused.
```

Comment:
Please capitalize, to keep in-line with the remainder of the function:

```suggestion
    // Write the BSP name into the clipmap struct so that it could be reused.
```